examples/src/clean-state.js

```diff
@@ -1,19 +1,23 @@
-import { NearBindgen, call, view, near } from "near-sdk-js";
+import { NearBindgen, call, view, near, encode, decode } from "near-sdk-js";
 
 @NearBindgen({})
 export class CleanState {
   @call({})
   clean({ keys }) {
-    keys.forEach((key) => near.storageRemove(key));
+    keys.forEach((key) => near.storageRemove(encode(key)));
```

Comment:
Developers should not do encoding on the contract level.

Reply:
Same issue as above: https://github.com/near/near-sdk-js/pull/308#discussion_r1045451673